test/create-server.spec.ts

```diff
@@ -63,6 +63,7 @@ describe("createServer", () => {
         },
         shouldEndSession: true,
       },
+      sessionAttributes: {},
```

Comment:
i'd rather we don't include `sessionAttributes` in the response if it's empty

Reply:
I agree, and tried to do it, but it was rather hard to make consistent. Basically the to do it I'd need to make the sessionAttributes undefined conditionally. I could do that in saveSession, but not all of the tests actually call saveSession. After messing with it for a while, I just decided that consistently there was better than sometimes there and empty, and sometimes not.